package on2013_07.on2013_07_24_Codeforces_Round__193.C___Students__Revenge;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2013_07/on2013_07_24_Codeforces_Round__193/C___Students__Revenge/C - Students' Revenge.task"))
			Assert.fail();
	}
}
